ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+      block:  ":oo-udp - [0:0]"
+
+    oo_udp_limit_chain:
+    - chain: oo-udp-limit
+      after: "filter$"
+      regex: :oo-udp-limit - \[0:0\]
+      block:  ":oo-udp-limit - [0:0]"
+
+    filtering_udp_rules:
+    # allow all outbound UDP for privileged pods
+    # send all other traffic to the oo-udp chain so that rules that allow
+    # traffic can call -j RETURN instead of -j ACCEPT, and thus permit us
+    # to do later accounting
+    - chain: OPENSHIFT-OUTPUT-FILTERING
+      after: "OPENSHIFT-OUTPUT-FILTERING"
+      regex: "id_outbound_udp_filtering_2_"
+      line: "-I OPENSHIFT-OUTPUT-FILTERING -m comment --comment id_outbound_udp_filtering_2_ -s {{ node_cidr }} -o eth0 -p udp -j oo-udp"
+
+    - chain: OPENSHIFT-OUTPUT-FILTERING
+      after: "id_outbound_udp_filtering_2_"
+      regex: "id_outbound_udp_filtering_1_"
+      line: "-I OPENSHIFT-OUTPUT-FILTERING -m comment --comment id_outbound_udp_filtering_1_ -s 172.0.0.0/8 -o eth0 -p udp -j ACCEPT"
+
+    filtering_udp_memory_rules:
+    - chain: OPENSHIFT-OUTPUT-FILTERING
+      regex: "id_outbound_udp_filtering_2_"
+      line: "-I OPENSHIFT-OUTPUT-FILTERING -m comment --comment id_outbound_udp_filtering_2_ -s {{ node_cidr }} -o eth0 -p udp -j oo-udp"
+
+    - chain: OPENSHIFT-OUTPUT-FILTERING
+      regex: "id_outbound_udp_filtering_1_"
+      line: "-I OPENSHIFT-OUTPUT-FILTERING -m comment --comment id_outbound_udp_filtering_1_ -s 172.0.0.0/8 -o eth0 -p udp -j ACCEPT"
+
+    outbound_udp_rules:
+    # Allow all DNS traffic to whitelisted DNS servers, with no limits.
+    # Limit any remaining permitted UDP traffic to 30 datagrams per second,
+    # and define permitted UDP traffic for un-privileged pods
+    # Finally, drop all other UDP traffic. Log dropped traffic up to once per minute.
+    # Because all dropped traffic is crammed into this single limited log rule,
+    # log entries will only be a sampling of actual dropped traffic.
+    - chain: oo-udp
+      after: "id_outbound_udp_filtering"
+      regex: "id_outbound_udp_rule_1_"
+      block: |-
+        -A oo-udp -d 172.16.0.0/12 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 204.13.250.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 204.13.251.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 208.78.70.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -d 208.78.71.23/32 -o eth0 -p udp -m udp --dport 53 -j RETURN
+        -A oo-udp -s 10.0.0.0/8 -o eth0 -p udp -j oo-udp-limit
+        -A oo-udp -o eth0 -p udp -m udp --dport 123 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 67 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 161 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 500 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 1514 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 4500 -j RETURN
+        -A oo-udp -o eth0 -p udp -m udp --dport 25826 -j RETURN
```

Comment:
I think it might be a good idea to allow some minimal DNS traffic to other DNS servers. The local DNS traffic isn't throttled, but this would be, so I don't think it's dangerous:
```
-A oo-udp -o eth0 -p udp -m udp --dport 53 -j RETURN
```